pkg/systemlogmonitor/log_monitor_test.go

```diff
@@ -145,11 +153,24 @@ func TestGenerateStatus(t *testing.T) {
 }
 
 func TestGoroutineLeak(t *testing.T) {
-	orignal := runtime.NumGoroutine()
+	original_prof := pprof.Lookup("goroutine")
```

Comment:
Should those changes to `TestGoroutineLeak` be moved to the other PR?

Reply:
Done.